app/javascript/psulib_blacklight/javascripts/availability.js

```diff
@@ -349,7 +361,7 @@ function availabilityDataStructurer(holdingMetadata) {
 
             // Supplement data with an itemType and remove callNumber conditionally
             holdingMetadata[libraryID].forEach(function(element) {
-                element.itemType = (element.itemTypeID in allItemTypes) ? allItemTypes[element.itemTypeID] : "";
+                element.itemType = (element.itemTypeID in allItemTypes) ? element.itemTypeID : "";
```

Comment:
@cdmo shouldn't this still be `allItemTypes[element.itemTypeID]`?

Reply:
Good catch, yes, sorry for the mistake

Reply:
```suggestion
                element.itemType = (element.itemTypeID in allItemTypes) ? allItemTypes[element.itemTypeID] : "";
```
per our conversation with @cdmo, reverting this back